### PR TITLE
Added app description to DetailsPanel

### PR DESCRIPTION
**Pull request by rhinoswirl**
Added display of description according to #12  and based on the package manifest update submitted in https://github.com/webosbrew/docs/pull/2.

### CODE REVIEW

**Comment by Informatic** [comment]
LGTM, we'll merge this when webosbrew/docs#1 gets in.

**Comment by Informatic** [comment]
LGTM (test)

File: frontend/views/DetailsPanel.js [hunk 1]
```diff
@@ -144,6 +148,15 @@ module.exports = kind({
     // This is data loaded from the web service
     {from: 'packageInfo.title', to: 'title'},
     {from: 'packageInfo.id', to: 'titleBelow'},
+    {
+      from: 'packageInfo.appDescription', to: '$.appDescription.content', transform: function (description) {
+        var appDescription = 'No description provided for this package'
+        if (description instanceof String && description.trim().length) {
```
Comment (Informatic) [inline frontend/views/DetailsPanel.js:154]: Out of curiosity - have you actually tested this yet? ;)

```
>>> "abc" instanceof String
 <- false
```

Comment (rhinoswirl) [reply frontend/views/DetailsPanel.js:154]: You are right. instanceof only works with new.
Correcting it.

**Comment by Informatic** [comment]
Note: seems like `BodyText` is the only component in Enyojs that defaults to `allowHtml: true`... I am pretty sure this is going to screw us at some point...

**Review by Informatic — APPROVED** [review]

**Pull request merged**